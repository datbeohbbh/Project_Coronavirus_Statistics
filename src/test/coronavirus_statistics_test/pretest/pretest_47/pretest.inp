50 100
q 1 46 213
? 6
? 4 1
q 1 43 332
? 4 1
q 1 33 223
q 1 5 339
q 1 8 415
? 4 4
? 4 3
q 2 43 1
q 1 13 413
q 1 37 425
? 7
q 1 18 361
q 1 23 38
? 7
q 3 18 29
q 1 10 50
? 6
q 1 21 213
q 2 10 21
q 2 23 27
q 2 21 57
? 4 11
? 4 1
q 1 19 487
? 4 7
q 1 36 35
q 3 36 21
q 2 5 107
q 2 23 6
? 6
? 4 4
q 2 5 223
q 3 10 15
q 2 8 7
? 4 8
q 1 49 133
q 2 18 320
? 4 14
q 3 46 136
q 1 19 203
? 4 9
? 4 7
q 1 25 105
q 3 5 6
q 3 18 11
q 2 10 2
? 4 4
? 4 10
? 6
? 4 7
? 4 8
? 4 12
q 3 18 1
? 4 13
? 7
q 2 33 66
q 3 8 55
q 2 13 337
? 4 4
q 2 19 125
q 2 5 2
q 1 34 265
q 2 8 41
? 4 3
q 2 21 79
? 7
? 7
q 2 8 19
? 4 11
q 2 46 11
q 1 22 89
? 4 10
? 4 12
q 1 6 218
q 1 30 263
? 7
q 1 32 354
? 7
? 7
? 4 5
? 7
? 4 16
? 4 5
? 4 8
? 4 7
? 6
? 4 8
? 6
? 4 18
? 7
? 4 10
? 7
? 6
? 4 11
? 4 17
? 4 19
? 7