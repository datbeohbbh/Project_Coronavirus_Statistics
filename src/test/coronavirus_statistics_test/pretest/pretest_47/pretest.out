Query ? 6 :
Currently, no recovery cases have been recorded

Query ? 4 1 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 213
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 213
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 5 
- Information of the city 5 -
-> Number of current patients : 339
-> Has recovered : 0
-> Was dead : 0

Query ? 4 3 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 332
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
Currently, no deaths have been recorded

Query ? 7 :
Currently, no deaths have been recorded

Query ? 6 :
The city with the highest recovery is city with the index 43
Quantity: 1 people

Query ? 4 11 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 425
-> Has recovered : 0
-> Was dead : 0

Query ? 4 1 :
The answer is the city with the index 23 
- Information of the city 23 -
-> Number of current patients : 11
-> Has recovered : 27
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 18 
- Information of the city 18 -
-> Number of current patients : 332
-> Has recovered : 0
-> Was dead : 29

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 107 people

Query ? 4 4 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 156
-> Has recovered : 57
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 331
-> Has recovered : 1
-> Was dead : 0

Query ? 4 14 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 487
-> Has recovered : 0
-> Was dead : 0

Query ? 4 9 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 223
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 4 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 12
-> Has recovered : 23
-> Was dead : 15

Query ? 4 10 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 223
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 5
Quantity: 330 people

Query ? 4 7 :
The answer is the city with the index 25 
- Information of the city 25 -
-> Number of current patients : 105
-> Has recovered : 0
-> Was dead : 0

Query ? 4 8 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 8 
- Information of the city 8 -
-> Number of current patients : 408
-> Has recovered : 7
-> Was dead : 0

Query ? 4 13 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 425
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 4 4 :
The answer is the city with the index 36 
- Information of the city 36 -
-> Number of current patients : 14
-> Has recovered : 0
-> Was dead : 21

Query ? 4 3 :
The answer is the city with the index 10 
- Information of the city 10 -
-> Number of current patients : 12
-> Has recovered : 23
-> Was dead : 15

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 4 11 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 265
-> Has recovered : 0
-> Was dead : 0

Query ? 4 10 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 4 12 :
The answer is the city with the index 34 
- Information of the city 34 -
-> Number of current patients : 265
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 4 5 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 66
-> Has recovered : 11
-> Was dead : 136

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 4 16 :
The answer is the city with the index 43 
- Information of the city 43 -
-> Number of current patients : 331
-> Has recovered : 1
-> Was dead : 0

Query ? 4 5 :
The answer is the city with the index 46 
- Information of the city 46 -
-> Number of current patients : 66
-> Has recovered : 11
-> Was dead : 136

Query ? 4 8 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 89
-> Has recovered : 0
-> Was dead : 0

Query ? 4 7 :
The answer is the city with the index 21 
- Information of the city 21 -
-> Number of current patients : 77
-> Has recovered : 136
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 337 people

Query ? 4 8 :
The answer is the city with the index 22 
- Information of the city 22 -
-> Number of current patients : 89
-> Has recovered : 0
-> Was dead : 0

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 337 people

Query ? 4 18 :
The answer is the city with the index 37 
- Information of the city 37 -
-> Number of current patients : 425
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 4 10 :
The answer is the city with the index 49 
- Information of the city 49 -
-> Number of current patients : 133
-> Has recovered : 0
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

Query ? 6 :
The city with the highest recovery is city with the index 13
Quantity: 337 people

Query ? 4 11 :
The answer is the city with the index 33 
- Information of the city 33 -
-> Number of current patients : 157
-> Has recovered : 66
-> Was dead : 0

Query ? 4 17 :
The answer is the city with the index 32 
- Information of the city 32 -
-> Number of current patients : 354
-> Has recovered : 0
-> Was dead : 0

Query ? 4 19 :
The answer is the city with the index 19 
- Information of the city 19 -
-> Number of current patients : 565
-> Has recovered : 125
-> Was dead : 0

Query ? 7 :
The city with the highest death is city with the index 46
Quantity: 136 people

